nixos/doc/manual/configuration/profiles/perlless.section.md

```diff
@@ -0,0 +1,11 @@
+# Perlless {#sec-perlless}
+
+::: {.warning}
+If you enable this profile, you will NOT be able to switch to a new
```

Comment:
What should be done instead?

Reply:
Right now, the only thing you can do is replace the entire system from "outside" the running system. This could be accomplished via partition based A/B updates (e.g. via systemd-sysupdate) or by copying Nix store paths.

Reply:
A tutorial documenting this to novices (like me) would be nice.

Reply:
Yes I'm planning on doing something along those lines but this is out of scope for this change.